ui-manchette/src/components/OperationalPointList.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import { OperationalPointType } from '../types/pathPropertiesTypes';
+import OperationalPoint from './OperationalPoint';
+import cx from 'classnames';
+import { positionKm } from './helpers';
+
+interface OperationalPointListProps {
+  operationalPoints: OperationalPointType[];
+  isProportionnal?: boolean;
+  zoom?: number;
+  width?: number;
+}
+
+const OperationalPointList: React.FC<OperationalPointListProps> = ({
+  operationalPoints,
+  isProportionnal = false,
+  zoom = 1,
+}) => {
+  const [operationalPointsToDisplay, setOperationalPointsToDisplay] = React.useState<
+    OperationalPointType[]
+  >([]);
+  const baseHeight = 32;
+
+  const operationalPointStyle = (op: OperationalPointType, nextOp: OperationalPointType) => {
```

Comment:
```suggestion
  const operationalPointStyle = (op: OperationalPointType, nextOp?: OperationalPointType) => {
```

Reply:
nextOp can be undefined in case of the last OP from the array

Reply:
I agree, I think you should specify in the function signature that currentOp can be undefined (also I prefer nextOp to currentOp since you give `operationalPointsToDisplay[index + 1]` l80)
```suggestion
  const operationalPointStyle = (op: OperationalPointType, nextOp?: OperationalPointType) => {
```